.github/workflows/ci-fork.yml

```diff
@@ -0,0 +1,62 @@
+name: CI (Fork)
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+
+jobs:
+  unit-test-results:
+    name: Unit Test Results
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.conclusion != 'skipped' &&
+      github.event.workflow_run.head_repository.full_name != github.repository
+
+    steps:
+      - name: Download Artifacts
```

Comment:
It's unfortunate that `actions/download-artifact` does not yet support specifying a different workflow ID. I think we should add a comment here referencing issue https://github.com/actions/download-artifact/issues/3 so we remember to switch to the official action once it's solved.